README.md

```diff
@@ -721,6 +721,21 @@ meant to be able to change with it.
     subject(:article) { FactoryBot.create(:some_article) }
     ```
 
+  * <a name="needed-data"></a>
+    Do not load more data than needed to test your code.
+    <sup>[[link](#needed-data)]</sup>
+
+    ```ruby
+    # good
+    RSpec.describe User do
+      describe ".top" do
+        before { FactoryBot.create_list(:user, 3) }
+
+        it { expect(described_class.top(2).size).to eq 2 }
```

Comment:
I don’t agree that a line like this is an example of a “good” test. I know that the point of the example is to show that we only build 3 user objects, but I would still recommend 1) extracting `described_class.top(2)` or `described_class.top(2).size` to a local variable, and 2) not use the “implicit” one-line `it` syntax, but add a string description to the `it` call.

Reply:
Right, this one seems to contradict at least a couple of the other rules.
WDYT of:
```
subject { described_class.top(2) }

it { is_expected.to have(2).items }
```
instead ?

Reply:
That indeed looks better!

Do we have a recommendation for or against the one-line `it` block syntax?

Reply:
Awesome, will update. 👍 

We do, a [quite recently added one](#40) [here](https://github.com/rubocop-hq/rspec-style-guide#it-and-specify). Appreciate any feedback on it as well.